examples/web_worker_example/build.yaml

```diff
@@ -23,7 +10,8 @@ targets:
           named_parameters: true
           new_sql_code_generation: true
       build_web_compilers|entrypoint:
+        enabled: true
         generate_for:
-          # This one is compiled in the other target
-          exclude:
-            - "web/worker.dart"
+          - web/worker.dart
+        options:
+          compiler: dart2js
```

Comment:
Was the previous build configuration breaking anything?

For debugging purposes, I prefer the main entrypoint to be compiled with dartdevc. The worker needs to be dart2js and that's fine, but if possible I want to use both compilers in a single build here.

Reply:
Ah I see what's going wrong now. I'll try to fix it with advice from https://github.com/dart-lang/build/issues/3275#issuecomment-1087697299.

Reply:
Fixed in 4fae017d364da6097dee7bdd50a620e541e0f3ff.

Reply:
Yes, this was the problem. Sorry, I didn't notice that this build file should generates two separate files